Source/WebCore/Modules/credentialmanagement/CredentialsContainer.cpp

```diff
@@ -144,6 +129,22 @@ void CredentialsContainer::preventSilentAccess(DOMPromiseDeferred<void>&& promis
     promise.resolve();
 }
 
+template<typename Options>
+bool CredentialsContainer::performCommonChecks(Options&& options, CredentialPromise& promise)
```

Comment:
It would be simpler to just pass a `const Options&` here. Then there's no need to move/forward at the call sites.

Reply:
Agree... and yeah, was silly I was poking at things after moving them.